rllib/core/models/configs.py

```diff
@@ -160,6 +160,13 @@ class _MLPConfig(ModelConfig):
             "_" are allowed.
         output_layer_bias_initializer_config: Configuration to pass into the
             initializer defined in `output_layer_bias_initializer`.
+        clip_log_std: If the log std should be clipped by `log_std_clip_param`.
```

Comment:
nit: I feel like this comment is confusing. We should write that clipping is only applied to those action distribution parameters that encode the log-std for a DiagGaussian action distribution. Any other node's output (or if there is no DiagGaussian) is not clipped.

Mentioning the value function makes it confusing.